use-cases/sonarqube_datasets/README.md

```diff
@@ -0,0 +1,124 @@
+# Zowe CLI SonarQube Sample
+This sample serves as a working example of using Zowe SDK and SonarQube to analyze source code contained in MVS data sets. 
+
+The sample includes a means to execute the analysis locally via `npm` scripts and via a Jenkins pipeline.
+
+The sample will download the source files from the partitioned data-sets specified in [custom_properties.json](./custom_properties.json) and run the `sonar-scanner` CLI to analyze the source with SonarQube.
+
+**NOTE:** After running the SonarScanner via `npm run sonar`, the downloaded source files will be deleted from the local directory.
+
+## Quick-Start
+For detailed information on configuration and execution, start with [Prerequisites](#Prerequisites).
+
+1. Clone the project
+2. Configure [custom_properties.json](./custom_properties.json)
```

Comment:
This links to a non-existent file. Should this readme include a step similar to the Assembler Metal readme?
> Make a copy of `properties.json` and name it `custom_properties.json`.

Reply:
Done